src/persist-txn/src/operator.rs

```diff
@@ -449,74 +469,6 @@ pub fn txns_data_shard_retry_params(cfg: &ConfigSet) -> RetryParameters {
     }
 }
 
-// NB: The API of this intentionally mirrors TxnsCache and TxnsRead. Consider
-// making them all implement the same trait?
-struct TxnsCacheTimely<T: Timestamp + Lattice + Codec64> {
-    name: String,
-    state: TxnsCacheState<T>,
-    input: AsyncInputHandle<T, Vec<(TxnsEntry, T, i64)>, Disconnected>,
-    buf: Vec<(TxnsEntry, T, i64)>,
-}
-
-impl<T> TxnsCacheTimely<T>
```

Comment:
🙌

Reply:
Heh, I thought this was so clever! But as always the best code is no code :D